distribution/changelog.txt

```diff
@@ -3,6 +3,7 @@
 - Feature: [#15642] Track design placement can now use contruction modifier keys (ctrl/shift).
 - Fix: [#21959] “Save this before...?” message does not appear when selecting “New Game”.
 - Fix: [#22231] Invalid object version can cause a crash.
+- Fix: [#22596] Land ownership fixes described by .parkpatch files are now only considered on scenarios.
```

Comment:
I would classify this as a change, rather than a fix.

Reply:
I was torn too, wrote as a change first and then changed to fix. Will apply later.